src/rgw/rgw_op.h

```diff
@@ -1714,6 +1714,7 @@ class RGWInitMultipart : public RGWOp {
   string upload_id;
   RGWAccessControlPolicy policy;
   ceph::real_time mtime;
+  std::unique_ptr<RGWObjTags> obj_tags;
```

Comment:
could this be a std::optional, avoiding an allocation from the free store?

Reply:
updated